Packs/FeedCrowdstrikeFalconIntel/Integrations/CrowdStrikeIndicatorFeed/CrowdStrikeIndicatorFeed.yml

```diff
@@ -46,10 +46,11 @@ configuration:
   type: 16
 - display: First fetch time
   name: first_fetch
-  required: false
+  required: true
   type: 0
-  additionalinfo: The time range to consider for the initial data fetch. Leave empty
-    to fetch from the first available indicator.
+  additionalinfo: The time range to consider for the initial data fetch.
+    Warning, This feed fetches about tens of thousands of indicators every day, Please consider this when setting this parameter to an earlier time to prevent system overload.
```

Comment:
```suggestion
    Warning: This feed fetches about tens of thousands of indicators per day. Please consider that when setting this parameter to an earlier in to the past, as it may overload the system with indicators.
```